internal/service/s3/object_copy_test.go

```diff
@@ -343,13 +445,13 @@ func testAccObjectCopyConfig_baseSourceAndTargetBuckets(sourceBucket, targetBuck
 resource "aws_s3_bucket" "source" {
   bucket = %[1]q
 
-  force_destroy = true
+  # force_destroy = true
```

Comment:
Observing a couple test failures which include this config and may be caused by this removal:

```
=== NAME  TestAccS3ObjectCopy_targetWithMultipleSlashes
    testing_new.go:90: Error running post-test destroy, there may be dangling resources: exit status 1

        Error: deleting Amazon S3 (Simple Storage) Bucket (tf-acc-test-2703616533666920534): BucketNotEmpty: The bucket you tried to delete is not empty
                status code: 409, request id: X9GWS1JDPA4ZGQ2P, host id: mz8v9CNqsDUTOWSRS88M5xSPDA0aalhsWsnKKgx6poBNP9AEpmmmumTwAaj5Lgya1zd0RsCME9g=
```

```
=== NAME  TestAccS3ObjectCopy_sourceWithSlashes
    testing_new.go:90: Error running post-test destroy, there may be dangling resources: exit status 1

        Error: deleting Amazon S3 (Simple Storage) Bucket (tf-acc-test-1881025664579207273): BucketNotEmpty: The bucket you tried to delete is not empty
                status code: 409, request id: 6WZAE710P2H71PJK, host id: UuaGZmGZ5oX04xzGf4FBtCd5MiE5WR/U4sSyHYFCPsuj44Qx8s7sZ0+snjyrGOu+BVx35cuDXZY=
```

Reply:
Will be fixed by addressing automatic URO cleaning with SDK v1.